SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -54,6 +45,8 @@ class USpatialStaticComponentView;
 class USpatialWorkerConnection;
 class USpatialWorkerFlags;
 
+DECLARE_DELEGATE(PostWorldWipeDelegate);
```

Comment:
What's this?

Reply:
Something that's defined in SnapshotManager, and that the NetDriver needs. I just reported it as a "forward declaration" of sorts.